### PR TITLE
dmytro.vakulenko #3

### PR DESCRIPTION
**Pull request by DrSmile444**
Finished Homework 3

### CODE REVIEW

File: lesson 3/dmytro.vakulenko/project.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  redis
+
+# Ruby
+docker run --rm -d \
+  --name ${RUBY_NAME} \
+  --net ${DOCKER_NETWORK} \
+  -e DB=${DB} \
+  -e CACHE=${CACHE} \
+  -p ${EXPOSE_PORT}:4567 \
+  --mount type=bind,source="${PROJECT_DIR}",target=/app \
```
Comment (MrCartoon) [inline lesson 3/dmytro.vakulenko/project.sh]: краще використати `--mount "${PROJECT_DIR}":/app`

**Review by MrCartoon — CHANGES_REQUESTED** [review]

File: lesson 3/dmytro.vakulenko/project.sh (hunk 1 above)
Comment (DrSmile444) [reply lesson 3/dmytro.vakulenko/project.sh]: working on it, thank you!

Comment (DrSmile444) [reply lesson 3/dmytro.vakulenko/project.sh]: Done! Review, please 😄

File: lesson 3/dmytro.vakulenko/project.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  redis
+
+# Ruby
+docker run --rm -d \
+  --name ${RUBY_NAME} \
+  --net ${DOCKER_NETWORK} \
+  -e DB=${DB} \
+  -e CACHE=${CACHE} \
+  -p ${EXPOSE_PORT}:4567 \
+  --mount "${PROJECT_DIR}":/app \
```
Comment (MrCartoon) [inline lesson 3/dmytro.vakulenko/project.sh:76]: Personally, I use `-v` (volume) instead of `--mount`.

**Review by MrCartoon — APPROVED** [review]

**Pull request merged**